02-Homework/script.js

```diff
@@ -0,0 +1,56 @@
+// Assignment Code
+var generateBtn = document.querySelector('#generate');
+
+function generatePassword() {
+  // PLEASE ADD ALL YOUR CODE HERE
+}
+alert ("text");
+var text
+var intiger
+if ( text = intiger) {
+"check is within range"
+} else {
+  alert and break
+}
+
+var confirm; (lowercase)
+var confirm; (uppercase)
+var confirm; (number)
+var confirm; (special)
+
+if (if all is false) {
+  alert and break
+
+} else {
+  return criteria
+}
+
+const (uppercase) "ABCDEFGHIJKLMNOPQRSTUVWXYZ"
```

Comment:
Variables should be defined like this:

`const uppercase = "ABCDEFGHIJKLMNOPQRSTUVWXYZ"`